portality/api/v2/data_objects/common_journal_application.py

```diff
@@ -0,0 +1,30 @@
+from copy import deepcopy
+
+from portality.lib import dataobj, swagger
+
+class OutgoingCommonJournalApplication(dataobj.DataObj, swagger.SwaggerSupport):
+
+    @classmethod
+    def from_model(cls, journal_or_app):
+        d = deepcopy(journal_or_app.data)
+
+        # we need to re-write the preservation section
+        # joa = d.get("bibjson", {}).get("preservation")
+        joa = journal_or_app.bibjson().preservation
```

Comment:
These kinds of translations should go away with the new version of the API, and we should stick to the core model format as much as possible.